src/components/grid/grid-item/grid-item.style.js

```diff
@@ -0,0 +1,63 @@
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+
+function responsiveGridItem({
+  responsiveSettings,
+  gridColumnStart,
+  gridColumnEnd,
+  gridRowStart,
+  gridRowEnd
+}) {
+  const item = responsiveSettings.map((setting) => {
+    const {
+      alignSelf,
+      colStart,
+      colEnd,
+      justifySelf,
+      maxWidth,
+      rowStart,
+      rowEnd
+    } = setting;
+
+    return css`
```

Comment:
alignment seems to be off here. probably worth adding 2 white spaces to entire content of css.